schemata/user.yaml

```diff
@@ -17,4 +17,8 @@ properties:
   avatar:
     type: string
     example: "http://example.net/user.png"
-required: [uuid, name]
+  canUse:
+    description: Feature flagging based on user's plan. Available flags are siteMediaCta, edCta, edEmbed
```

Comment:
this is the wrong way to do it, actually add the properties to the object with types, descriptions etc.